test/browser-providers.js

```diff
@@ -19,15 +19,15 @@ const browserConfig = {
   'Android4.2':   { unitTest: {target: null, required: false }},
   'Android4.3':   { unitTest: {target: null, required: false }},
   'Android4.4':   { unitTest: {target: null, required: false }},
-  'Android5':     { unitTest: {target: 'SL', required: false }},
+  'Android5':     { unitTest: {target: null, required: false }},
   'Safari7':      { unitTest: {target: null, required: false }},
   'Safari8':      { unitTest: {target: null, required: false }},
-  'Safari9':      { unitTest: {target: 'BS', required: false }},
-  'Safari10':     { unitTest: {target: 'SL', required: false }},
+  'Safari9':      { unitTest: {target: null, required: false }},
+  'Safari10':     { unitTest: {target: null, required: false }},
```

Comment:
We should be running Safari 9 and 10; I didn't realize these were not marked as required